### PR TITLE
Metrics api updates

### PR DESCRIPTION
**Pull request by siddtewari**

### CODE REVIEW

File: metrics_api.md [hunk 1]
```diff
@@ -42,7 +47,11 @@ Each dataset is a hash consisting of metadata and an array of values. Each eleme
 }
 ```
 
-The metadata hash describes the data you are sending. It consists of an asset type, the granularity of the included data, and an array of keys that describes the 'type' of each element in the values array.
+The **metadata hash** describes the data you are sending. It consists of:
+
+ - an asset type, 
+ - the granularity of the included data, and 
+ - an array of keys that describes the 'type' of each element in the values array.
```
Comment (dickwallace01) [inline metrics_api.md]: You are not consistent about ending a list w/ period - see above.

File: metrics_api.md [hunk 2]
```diff
@@ -57,54 +66,70 @@ The metadata hash describes the data you are sending. It consists of an asset ty
...
+The following **keys** are supported:
+
+- For all asset types:
+
+  - assetId
+  - timestamp
+
+- For aws:ec2:instance assets:
+
+	- cpu:used:percent.avg
```
Comment (dickwallace01) [inline metrics_api.md]: Why is the indentation deeper than above?

File: metrics_api.md (hunk 1 above)
Comment (siddtewari) [inline metrics_api.md]: addressed!

File: metrics_api.md (hunk 2 above)
Comment (siddtewari) [inline metrics_api.md]: addressed!

File: metrics_api.md [hunk 3]
```diff
@@ -340,20 +365,20 @@ You can limit what time range to retrieve asset information for.  The default is
 
 - mtd
 - last_month
-- last_3_months
-- last_6_months
-- last_12_months
+- last\_3_months
```
Comment (dickwallace01) [inline metrics_api.md]: Shouldn't you just quote these as well?

File: metrics_api.md [hunk 4]
```diff
@@ -57,72 +66,84 @@ The metadata hash describes the data you are sending. It consists of an asset ty
...
+- `cpu:used:percent.max`
+- `cpu:used:percent.min`
+- `memory:free:bytes.avg`
+- `memory:free:bytes.min`
+- `memory:free:bytes.max`
+- `memory:size:bytes.avg`
+- `memory:size:bytes.min`
+- `memory:size:bytes.max`
+- `memory:used:percent.avg`
+- `memory\:used:percent.min`
```
Comment (dickwallace01) [inline metrics_api.md]: Do you still need the \ before the colon?

**Comment by dickwallace01** [comment]
:+1:  w/ a couple more comments

File: metrics_api.md (hunk 3 above)
Comment (siddtewari) [inline metrics_api.md]: The escape character does not work for colon but works for underscore

File: metrics_api.md (hunk 4 above)
Comment (siddtewari) [inline metrics_api.md]: fixed

**Pull request merged**